### PR TITLE
[IMPROVEMENT] Added database upgrade mechanism

### PR DESCRIPTION
**Pull request by T1duS**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [X] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that another pull request for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [X] I am an active contributor to the project.

---

I have added database upgrade mechanism using alembic.
One can install it by pip install alembic
To migrate simply give the following command - alembic upgrade head

### CODE REVIEW

**Comment by canihavesomecoffee** [comment]
I'm missing instructions on how to use this ;)

File: README.md [hunk 1]
```diff
@@ -17,14 +17,19 @@ reported through a variety of channels (private email, mailing list, GitHub,
...
-everyone ([after signing up](https://sampleplatform.ccextractor.org/account/signup)), that provides a single place to upload, view 
+everyone ([after signing up](https://sampleplatform.ccextractor.org/account/signup)), that provides a single place to upload, view
 samples and associated test results.
 
 ## Installation
 
 An installation guideline can be found here:
 [installation guide](install/installation.md).
 
+## migration
```
Comment (aadibajpai) [inline README.md]:
```suggestion
## Migration
```

**Comment by satyammittal** [comment]
@T1duS @saurabhshri we have to fix this first: https://github.com/CCExtractor/sample-platform/issues/254, which will check the constraint in this issue.

File: alembic/README [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Generic single-database configuration.
+
+# Usage
+
+For adding a table, update [env.py](env.py) and import the table class.
+
+To do database migration, simply type
+`alembic upgrade head`
```
[the remark this answers is not in the document]
Comment (MatejMecka) [reply alembic/README]: BTW: You are missing the .md file extension :))

[the remark this answers is not in the document]
Comment (T1duS) [reply alembic/README]: @MatejMecka `alembic upgrade head` works too

**Comment by thealphadollar** [comment]
@T1duS @canihavesomecoffee Please look into the PR.

**Pull request closed without merging**